CHANGELOG.md

```diff
@@ -2,6 +2,38 @@
 
 This document includes a curated changelog for each release. We also publish a changelog as the description of a [GitHub release](https://github.com/nginxinc/nginx-kubernetes-gateway/releases), which, by contrast, is auto-generated and includes links to all PRs that went into the release.
 
+## Release 0.2.0
```

Comment:
We need this change in the `main` branch in addition to the release branch. Would it make sense to create a second PR with this change that targets `main` and then cherry-pick the commit to the release branch?

Reply:
the PR to the main branch needs to be merged after this one is merged and after the release is created on GitHub, so that we don't promote 0.2.0 before we actually release it. The PR to the main will become a cherry-pick from the release branch. 
What to do you think?

Reply:
Sounds good!